packages/frontend/pages/positions.tsx

```diff
@@ -140,6 +143,12 @@ export function Positions() {
   } = usePositions()
 
   const { index, getCollatRatioAndLiqPrice } = useController()
+  const { transactions } = useTransactionHistory()
+
+  const lastDeposit = useMemo(
+    () => transactions.find((transaction) => transaction.transactionType === TransactionType.MINT_SHORT),
```

Comment:
Do you only take the first mint-short transaction we find in the transaction array into account? What if user mint and then short multiple times? We only get the eth price at the last deposit timestamp to calculate collateral pnl?

Reply:
Ah yeah good point, also if a user mints+shorts multiple times

Reply:
In that case, thinking we take the average

Reply:
What do you mean by collateral pnl?

Reply:
Oh Im mean getting multiple timestamps if user having multiple mint+short txs